plm/tests/test_plm.py

```diff
@@ -177,8 +199,11 @@ def start_plm_service(host):
     return start_plm
 
 def get_git_commit():
-    headers = {'Authorization': 'token ' + os.environ.get("MONGO_REPO_TOKEN")}
-    url = f"https://api.github.com/repos/percona/percona-link-mongodb/commits/release-{version}"
+    headers = {'Authorization': 'token ' + str(os.environ.get("MONGO_REPO_TOKEN"))}
+    if install_repo == "release":
+        url = f"https://api.github.com/repos/percona/percona-link-mongodb/commits/release-{version}"
+    else:
+        url = f"https://api.github.com/repos/percona/percona-link-mongodb/commits/main"
```

Comment:
what if install_repo is testing?... please revert this change, as we usually check packages built using release branch anyway, if some reason it's required to do otherwise, it's ok for test to fail, or add xfail on commit check